tests/lib/in_memory/engine.py

```diff
@@ -65,7 +65,7 @@ def visit_Code(self, node):
         assert False
 
     def visit_Value(self, node):
-        return node.value
+        return Column({patient: [node.value] for patient in self.all_patients})
```

Comment:
This seemed too easy as a fix for bare `Value`s not being allowed as variables (and hence, see following commit) for simplifying `use_unrestricted_population()`. But it works...

Reply:
I should think most things seem too easy after everything else you've put yourself through in this PR...